clients/src/main/java/org/apache/kafka/clients/admin/CreateAclsResult.java

```diff
@@ -41,7 +41,7 @@
      * Return a map from ACL bindings to futures which can be used to check the status of the creation of each ACL
      * binding.
      */
-    public Map<AclBinding, KafkaFuture<Void>> results() {
+    public Map<AclBinding, KafkaFuture<Void>> value() {
```

Comment:
It's a good question. I was thinking of this as the `value` of the result (the same way a `Future` has a `value` whether it's a map or a scalar) and `all` would be the value transformed with `allOf`. But perhaps `values` fits better since this is a concrete method instead of a generic one.